indico/web/client/js/react/components/PersonDetailsModal.jsx

```diff
@@ -129,7 +129,9 @@ export default function PersonDetailsModal({
       )}
       <Form.Group widths="equal">
         <Form.Field>
-          <Translate as="label">Title</Translate>
+          <Translate as="label" context="Salutation, e.g., 'Mr.' or 'Mrs.'">
```

Comment:
I'd ditch the `, e.g., 'Mr.' or 'Mrs.'` part, it just makes it very verbose and easy to not end up with the exact same context in multiple places...

Reply:
I thought about it but wasn't sure if just 'Salutation' is clear enough. Overall I feel it's better to have a more verbose context even at the risk of them not being the exact same, but if you think it's enough I can simplify it